packages/compass-components/src/components/modals/modal-body.tsx

```diff
@@ -33,6 +34,10 @@ const variantStyle = {
   }),
 };
 
+const darkModeStyle = css({
+  paddingBottom: spacing[3],
```

Comment:
… but why do we want padding to differ between light mode and dark mode? This does look a bit odd in dark mode now because the bottom and top paddings aren’t the same anymore

Reply:
that's actually a fix for that, LG adds a border on the footer in darkmode